helm/alfresco-content-services/charts/alfresco-search/values.yaml

```diff
@@ -99,7 +99,7 @@ global:
     # https: to use mTLS auth (require appropriate certificate configuration)
     auth: secret
     # -- Shared secret to authenticate repo/solr traffic
-    #sharedsecret:
+    sharedsecret: null
```

Comment:
@alxgomz why this was commented?

Reply:
No good reason... It's not even consistent with the main value file.